delft3dworker/views.py

```diff
@@ -242,13 +255,70 @@ def start(self, request, pk=None):
 
         return Response(serializer.data)
 
+    @detail_route(methods=["get","post"])
+    def publish_company(self, request, pk=None):
+        scene = self.get_object()
+        groups = [group for group in self.request.user.groups.all() if "world" not in group.name]
+        print(groups)
+        # If we can still edit, scene is not published
+        published = not self.request.user.has_perm('delft3dworker.change_scene', scene)
+
+        if not published:
+
+            # Remove write permissions for user
+            remove_perm('add_scene', self.request.user, scene)
```

Comment:
I'm confused by this: how can 'add new instance of X' be assigned to object x1? And do we really want to revoke this permission?